src/openrct2/scripting/ScriptEngine.cpp

```diff
@@ -771,6 +777,44 @@ DukValue ScriptEngine::PositionToDuk(const CoordsXYZ& position)
     return DukValue::take_from_stack(ctx);
 }
 
+constexpr static const char* ExpenditureTypes[] = { "ride_construction",
+                                                    "ride_runningcosts",
+                                                    "land_purchase",
+                                                    "landscaping",
+                                                    "park_entrance_tickets",
+                                                    "park_ride_tickets",
+                                                    "shop_sales",
+                                                    "shop_stock",
+                                                    "food_drink_sales",
+                                                    "food_drink_stock",
+                                                    "wages",
+                                                    "marketing",
+                                                    "research",
+                                                    "interest" };
+
+std::string_view ScriptEngine::ExpenditureTypeToString(ExpenditureType expenditureType)
+{
+    auto index = static_cast<size_t>(expenditureType);
+    if (index >= 0 && index < std::size(ExpenditureTypes))
```

Comment:
A `size_t` is always bigger than or equal to 0. This is also why the CIs fail.

Reply:
Sorry index was previously an `int32_t` when I wrote the condition.